gui/services/forms.py

```diff
@@ -1823,6 +1823,22 @@ def clean_iscsi_target_authgroup(self):
         return int(group)
 
 
+class TargetExtentDelete(Form):
+    '''
+    Delete form for Targets/Associated Targets
+    '''
+    def __init__(self, *args, **kwargs):
+        self.instance = kwargs.pop('instance', None)
+        super(TargetExtentDelete, self).__init__(*args, **kwargs)
+        if not self.data:
+            connected_targets = notifier().iscsi_connected_targets()
+            # Get target name from targetname/extentname combination
+            target_to_be_deleted = str(self.instance).split(' / ')[0]
```

Comment:
Getting the str representation of self.instance and splitting it looks really weird? How about using direct instance fields?

self.instance.iscsi_whatever